### DISCUSSION TITLE
Request

### DISCUSSION THREAD

**Discussion by SilverStone391** [jhassan8/crunchyroll-tizen#75] [General]
Hi @jhassan8 could you make an unofficial crunchyroll app for crunchyroll Version 1.34 of the PS4 app, myself and many others would love to get this version of the Crunchyroll UI back. the User interface of this ver of crunchyroll looks exactly how it is in the attached file.  
![l8EZmOp](https://github.com/jhassan8/crunchyroll-tizen/assets/155844862/02e6b7ea-7695-4f98-bf47-dfd795f1d5f2)

**Comment by jhassan8** [comment]
Hi @SilverStone391, sorry, I didn't understand correctly, are you talking about an app for PS4 or an app for Samsung with the interface in the image? Thanks

**Reply by SilverStone391** [reply]
Sorry for the unclear message, I was talking about a Crunchyroll app for PS4,a jailbroken PS4 to be exact.
